vignettes/col_types.Rmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Column type guessing"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Column type guessing}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+library(readr)
```

Comment:
I have a personal rule that all code critical to making things work should be visible. Let's pull the `library(readr)` call out into a user-visible chunk.

`usethis::use_vignette()` will initiate a new vignette with the recommended two "starter" chunks. You could call it once to create a dummy vignette, see what it does, emulate that here (and delete the dummy vignette).

Reply:
I did `usethis::use_vignette()` but then I saw the other vignettes had `library(readr)` in the `include = FALSE` code chunk at the top so I moved it up there. I'll move it down so readers can see it.